configs/neptune.yaml

```diff
@@ -54,6 +54,8 @@ parameters:
   installments__last_k_trend_periods: '[10, 50, 100, 500]'
   installments__last_k_agg_periods: '[1, 5, 10, 20, 50, 100]'
   installments__last_k_agg_period_fractions: '[(5,20),(5,50),(10,50),(10,100),(20,100)]'
+  pos_cash__last_k_trend_periods: '[6, 12]'
+  pos_cash__last_k_agg_periods: '[6, 12, 30]'
   application_aggregation__use_diffs_only: True
```

Comment:
@Ninoko has this turned out to be best when set to `False` ?

Reply:
There is no meaningful difference, but computationally more expensive when `False`

Reply:
You mean application_aggregation__use_diffs_only?

Reply:
Yes